tools/osbuild-mpp

```diff
@@ -969,13 +969,19 @@ class Image:
     def from_dict(cls, js):
         size = js["size"]
         data = js["table"]
+        sector_size = js.get('sector_size', 512)
 
         with tempfile.TemporaryDirectory() as tmp:
             image = os.path.join(tmp, "disk.img")
             subprocess.run(["truncate", "--size", size, image], check=True)
-
-            table = PartitionTable.from_dict(data)
-            table.write_to(image)
+            cp = subprocess.run(["losetup", "--find", "--show", f"--sector-size={sector_size}", image],
+                                stdout=subprocess.PIPE, check=True)
```

Comment:
I looked into using the existing code for this that exists in osbuild but it wasn't straightforward how I would just call it here and this seemed simple enough.

Let me know if someone has better guidance here.